Methods/Parameters/Inventory/Inventory/Program.cs

```diff
@@ -0,0 +1,314 @@
+﻿using Inventory;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Xml.Linq;
+using System.Xml.Serialization;
+
+
+
+List<Item> items = new List<Item>
+{
+new Item { Name = "Apple", Quantity = 10, Price = 10 },
+new Item { Name = "Orange", Quantity = 10, Price = 10 },
+new Item { Name = "Lemon", Quantity = 10, Price = 10 },
+new Item { Name = "Pineapple", Quantity = 10, Price = 10 },
+new Item { Name = "Carrot", Quantity = 10, Price = 10 }
+
+};
+
+
+int n = 0;
+int totalvalue;
+int choice = 0;
+bool gameon = true;
+
+
+
+//int choice = 0;
+//char choice2 = 'x';
+//int updatechoice = 0;
+//int updatedvalue = 0;
+//string item = null;
+//int Quantity = 0;
+//int Price = 0;
+
+while (gameon) {
+
+    Console.Clear();
+    MainMenu();
+    while (!int.TryParse(Console.ReadLine(), out choice))
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number (1-7)");
+    }
+    Console.Clear();
+
+switch (choice)
+{
+    case 1:
+        ViewInventory();
+        break;
+    case 2:
+           AddItemDisplay();
+            //var (name, quantity, price) = AddItemDisplay();
+            //Additem(name, quantity, price);
+            break;
+    case 3:
+            RemoveItem();
+        break;
+    case 4:
+        UpdateItem();
+        break;
+    case 5:
+        SearchItem();
+        break;
+    case 6:
+        ViewTotal();
+        break;
+    case 7:
+            Console.Clear();
+            Console.WriteLine("Goodbye.");
+        gameon = false;
+        break;
+    default:
+        break;
+}
+}
+
+void CalculateTotal()
```

Comment:
This method directly modifies the totalvalue field. Consider returning the total value as an integer to make it more functional and predictable.